tasks/internal_reconcile_report_orphan/internal_reconcile_report_orphan.py

```diff
@@ -26,6 +26,13 @@
 PAGE_SIZE = 100
 
 
+with open("schemas/input.json", "r") as raw_schema:
+    schema = json.loads(raw_schema.read())
+validate_input = fastjsonschema.compile(schema)
```

Comment:
Since `validate_input` and `validate_output` are now global consider capitalizing them and potentially make them private by adding an `_` in front like `_VALIDATE_INPUT`.